File: apps/super-admin/src/lib/vehicle-image.ts
export function vehicleImageSrc(vehicleType: string): string {
  const t = vehicleType.toLowerCase();
  if (t.includes("bike") || t.includes("motor")) return "/img/vehicles/bike.jpg";
  if (t.includes("scoot")) return "/img/vehicles/bike.jpg";
  if (t.includes("sedan") || t.includes("4")) return "/img/vehicles/car.jpg";
  return "/img/vehicles/car.jpg";
}

File: apps/valet-operator-android/app/src/main/java/ai/instapark/valet/ui/util/VehicleImage.kt
package ai.instapark.valet.ui.util

import ai.instapark.valet.R
import androidx.annotation.DrawableRes

/**
 * Shared vehicle-type -> thumbnail mapping (Dashboard's "Next up" card, Vehicles, Queue,
 * Search results). Only bike/car photographic thumbnails exist -- scooter reuses the bike
 * photo, sedan/suv reuse the car photo (closest visual match), matching how all four call
 * sites already collapse vehicle type into this same 4-way grouping.
 */
@DrawableRes
fun vehicleImageRes(vehicleType: String): Int = when {
    "bike" in vehicleType.lowercase() || "scoot" in vehicleType.lowercase() -> R.drawable.img_thumb_bike
    else -> R.drawable.img_thumb_car
}
